src/gen_meta.ml

```diff
@@ -107,10 +107,10 @@ let gen_lib pub_name lib ~version =
              ]
            ; match Lib.kind lib with
            | Normal -> assert false
-           | Ppx_rewriter ->
+           | Ppx_rewriter _ ->(* TODO: Deal with args *)
```

Comment:
I don't think there is much to do here. Supporting the cookies would require changing `ocamlfind`